src/solver/conv_asm_1x1u.cpp

```diff
@@ -667,27 +666,26 @@ ConvSolution ConvAsm1x1U::GetSolution(const ConvolutionContext& params,
     GenerateClangDefsym(options, "ROCM_METADATA_VERSION", params.rmv.UseV3() ? 5 : 4);
 
     const PerformanceConfigConvAsm1x1U* pcfg = &config;
+
+    // Try to load config from environment variable
     PerformanceConfigConvAsm1x1U fromEnv;
-    if(!disableConfigOverrideFromEnv)
```

Comment:
[Recommendation] Remove `if` but keep the block in order to hide inner objects.

Reply:
Code blocks have been reverted

Reply:
@averinevg Please mark both my review comments as "resolved" (I can't do that)